src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -830,6 +832,9 @@ private void RaiseUpdateFrame()
             // Calculate delta and run frame.
             var delta = _updateStopwatch.Elapsed.TotalSeconds;
 
+            // Calculate how much the delta differs from the desired updatePeriod.
+            _updateTimeDeficit += delta - _updatePeriod;

```

Comment:
Can you please ensure these variables are zeroed as part of the Reset call?

Reply:
Done.